daliuge-translator/dlg/dropmake/lg.graph.schema

```diff
@@ -42,7 +42,7 @@
                     "type": "string"
                 },
                 "lastModifiedDatetime": {
-                    "type": "integer"
+                    "type": "number"
```

Comment:
These schema changes will need to be added to the schema file in: https://github.com/ICRAR/EAGLE_test_repo.

Reply:
Good catch! Eagle uses the schema in EAGLE_test_repo too.

Can we remove this schema file from the repo?

Reply:
The translator currently checks that a graph conforms to a schema. I think that the check no longer necessary, as the translator is now tested against a schema so errors will be raised ahead of time through either the EAGLE or DALiuGE tests. 

@awicenec would you be happy moving these changes to the EAGLE_test_repo, and instead using this PR to remove the translator schema check?

Reply:
As discussed, since it is possible to submit a graph directly to the translator, the schema verification needs to happen at run-time, not only during testing and for that we should not rely on a schema residing in a remote, separate repo. I've checked that JSON schema verification in Python supports accessing remote schemas, thus we could make this a bit more fancy.